pkgs/games/snekim/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, nimPackages, fetchFromGitea, xorg, raylib }:
+
+nimPackages.buildNimPackage rec {
+  pname = "snekim";
+  version = "1.2.0";
+
+  nimBinOnly = true;
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "annaaurora";
+    repo = "snekim";
+    rev = "v${version}";
+    sha256 = "sha256-Qgvq4CkGvNppYFpITCCifOHtVQYRQJPEK3rTJXQkTvI=";
+  };
+
+  buildInputs = [
```

Comment:
Please make these `propagatedBuildInputs` of `nimraylib-now`, unless you have a reason not to.

Reply:
@adamlwgriffiths unless you think the xorg stuff should go into the `raylib` `propagatedBuildInputs`.

Reply:
I'm not well versed enough in Nix to say for sure, I know enough to be dangerous.
But from my naive glance at the difference, it seems like a good suggestion.

Reply:
I read the nix manual (https://nixos.org/manual/nixpkgs/unstable/#ftn.id-1.4.2.5.5.2, and some other positions) but I still don't understand the difference between the buildInputs. I think uh… buildInputs is only for build-time inputs and propagatedBuildInputs is for build-time and run-time. Is that right?

Reply:
I set `stricDeps = true` in the latest force push and it still works so, it's all only needed at build-time?

Reply:
@ehmry

Reply:
Actually, this package doesn't need to specify xorg dependencies because raylib's package already does so.